kubernetes-itests/pom.xml

```diff
@@ -17,44 +17,37 @@
 
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
   <parent>
     <artifactId>kubernetes-client-project</artifactId>
     <groupId>io.fabric8</groupId>
     <version>4.7-SNAPSHOT</version>
-    <relativePath>../pom.xml</relativePath>
   </parent>
-  <modelVersion>4.0.0</modelVersion>
 
   <artifactId>kubernetes-itests</artifactId>
-  <name>Fabric8 :: Kubernetes :: Regression :: Tests</name>
   <packaging>jar</packaging>
-
-  <url>http://maven.apache.org</url>
-
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <junit.version>4.12</junit.version>
-  </properties>
+  <name>Fabric8 :: Kubernetes :: Regression :: Tests</name>
 
   <dependencies>
     <dependency>
```

Comment:
I know this change was not intentional but it has disabled integration tests running on CircleCI. Arquillian is not compatible with Junit 5 yet.

Reply:
Actually it was intentional, the `junit-jupiter-migrationsupport` transitively has the Junit 4.12 dependency, so probably I broke something else since Junit 4 is still there.

I will try to find some time to look at this, but I have no experience with arquillian.

Reply:
no problem. I've reverted this change in a separate PR for this module.

Reply:
I see the issue now, yes this change effectively disabled integration test from Arquillian :disappointed: